.github/workflows/main.yml

```diff
@@ -24,10 +24,9 @@ jobs:
       #----------------------------------------------
       #          install and run linters
       #----------------------------------------------
-      - run: python -m pip install black flake8 isort
+      - run: python -m pip install black flake8
       - run: flake8 .
       - run: black . --check
-      - run: isort .
```

Comment:
No more `isort`? 😢

Reply:
I was hitting conflicts with Black, so black would reformat the imports one way that isort would not like, then isort would format them another that black wouldn't like.

I removed isort in anger, tho I should have searched for a way of reconciling them :D

Reply:
Aaah, I hit that as well! You can add

```
[tool.isort]
profile = "black"
```

to `pyproject.toml`.

Reply:
oh great. thanks!